loopchain/blockchain/blockchain.py

```diff
@@ -317,7 +316,51 @@ def __add_block(self, block: Block, confirm_info, need_to_write_tx_info=True, ne
 
             return True
 
-    def __write_block_data(self, block: Block, confirm_info):
+    def __add_tx_to_block_db(self, block, receipts, batch=None):
```

Comment:
Changinge name is needed. It should be similar to 'write_block_data'.
Maybe 'write_transtions'? Remove 'block_db'

Reply:
I just keep it as a same before. I will consider it later refactoring.